client/state/stats/chart-tabs/reducer.js

```diff
@@ -25,6 +25,10 @@ export const counts = withSchemaValidation(
 		keyedReducer( 'period', ( state = [], action ) => {
 			switch ( action.type ) {
 				case STATS_CHART_COUNTS_RECEIVE: {
+					if ( action.data.length !== state.length ) {
```

Comment:
This workaround has been needed to prevent new days from being appended to the 7 days displayed on My Home when navigating to Stats:
![image](https://user-images.githubusercontent.com/1233880/80390961-6e54ea80-88ad-11ea-9dfd-532b564c858e.png)

Reply:
Worth leaving a comment for folks

Reply:
58b7d96